lib/compat/wordpress-6.5/interactivity-api/interactivity-api.php

```diff
@@ -178,3 +178,23 @@ function wp_interactivity_data_wp_context( array $context, string $store_namespa
 		'\'';
 	}
 }
+
+if ( ! function_exists( 'data_wp_context' ) ) {
+	/**
+	 * `data_wp_context()` was renamed to follow WordPress Core naming schemes.
+	 *
+	 * @link https://github.com/WordPress/gutenberg/pull/59465/
+	 * @link https://core.trac.wordpress.org/ticket/60575
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param array  $context         The array of context data to encode.
+	 * @param string $store_namespace Optional. The unique store namespace identifier.
+	 * @return string A complete `data-wp-context` directive with a JSON encoded value representing the context array and
+	 *                the store namespace if specified.
+	 */
+	function data_wp_context( array $context, string $store_namespace = '' ): string {
+		_deprecated_function( __FUNCTION__, '5.6.0', 'wp_interactivity_data_wp_context()' );
```

Comment:
The version should be '6.5'

Reply:
I'm not sure if we should include here the Gutenberg version when it was deprecated. As, in WP, it has only been in a couple of release candidates.

Reply:
Oop, read it backwards.  D'oh.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/59834/commits/e80cd37c67b3d1cf2fb7dbc639bb579d8a42c149

Unless we'd like to change it to refer to the Gutenberg release.

Reply:
No problem, is not a critical issue.